aziotctl/aziotctl-common/test-files/apply/dps-x509-pkcs11-est-bootstrap/config.toml

```diff
@@ -23,5 +23,9 @@ password = "estpwd"
 bootstrap_identity_cert = "file:///var/secrets/est-bootstrap-id.pem"
 bootstrap_identity_pk = "pkcs11:slot-id=0;object=est-bootstrap-id?pin-value=1234"
 
+[cert_issuance.est.identity_auto_renew]
```

Comment:
I thought we were planning to have a auto_renew explicit flag as well..

Reply:
I did that at first, but I think this way looks better.
- threshold and retry have no effect if auto_renew is false. This way, those settings don't exist when the [auto_renew] section isn't set.
- This allows the setting to be called something other than just `auto_renew`. So you can distinguish, e.g. `identity_auto_renew`.

If you agree, I'll update the doc for this.